flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/source/assigner/FutureNotifier.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please fix the apache header

Reply:
In every file

Reply:
`spotlessApply` should fix header problem

Reply:
Done.